fetch.bs

```diff
@@ -4621,7 +4621,11 @@ steps:
   <a for="fetch params">request</a>'s <a for=request>client</a> is a <a>secure context</a>, then set
   <var>timingInfo</var>'s <a for="fetch timing info">server-timing headers</a> to the
   result of <a for="header list">getting, decoding, and splitting</a> `<code>Server-Timing</code>`
-  from <var>response</var>'s <a for=response>header list</a>.
+  from <var>response</var>'s <a for="filtered response">internal response</a>'s
+  <a for=response>header list</a>.
```

Comment:
We're sure _response_ is always a filtered response at this point, right?

Reply:
It could be a standard CORS/same-origin response. Would that still be a filtered response? Is there a shorthand for receiving the "internal response if it's filtered or the regular response otherwise"?

Reply:
Step 14.2 of main fetch seems to ensure there's either a network error or a filtered response so I think we're good.

HTML defines "unsafe response" which we should maybe define here as well, but would be good to know where it's needed first.